srcpkgs/colord-kde/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'colord-kde'
+pkgname=colord-kde
+version=0.5.0
+revision=1
+build_style=cmake
+hostmakedepends="kcoreaddons extra-cmake-modules gettext pkg-config qt5-host-tools qt5-qmake"
+makedepends="kcmutils-devel libXrandr-devel lcms2-devel plasma-framework-devel"
+depends="colord"
+short_desc="KDE colord integration"
+maintainer="Danila Kryukov <danila.kryukov.dev@yandex.ru>"
+license="GPL-2.0-or-later"
+homepage="https://invent.kde.org/graphics/colord-kde"
+distfiles="${KDE_SITE}/colord-kde/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=6c545009af737ad67900e4d8549086bc65b979c799b2402871bba44addb52d2f
+
+if [ "$CROSS_BUILD" ]; then
```

Comment:
Wasn't it building fine without?

Reply:
Remove this if, build correctly, but in time when CI build my package qt5 updating to 5.15 and CI build qt5 not my pkg. how to restart Travis check?

Reply:
You can just rebase the commits on upstream and force-push them.